### PR TITLE
added sata drive not showing  up in disk utility

### PR DESCRIPTION
**Pull request by gmachina**
use sata mode ahci

### CODE REVIEW

File: troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -302,7 +302,7 @@ Outdated OpenRuntime.efi, make sure BOOTx64.efi, OpenCore.efi and OpenRuntime ar
 * [`kextd stall[0]: AppleACPICPU`](../troubleshooting/troubleshooting.md#kextd-stall0-appleacpicpu)
 * [MediaKit reports not enough space](../troubleshooting/troubleshooting.md#mediakit-reports-not-enough-space)
 * [DiskUtility failing to erase](../troubleshooting/troubleshooting.md#diskutility-failing-to-erase)
-
+* [SATA Drives Not Shown in Disk Utility](../troubleshooting/troubleshooting.md#diskutility-not-showing-sata-drives)
```
Comment (khronokernel) [inline troubleshooting/troubleshooting.md:305]: This doesn't match with the header, the hyperlink will fail in this case. Should be this:
```
../troubleshooting/troubleshooting.md#sata-drives-no-shown-in-disk-utility
```

**Comment by gmachina** [comment]
Ok

**Comment by khronokernel** [comment]
Added a comment, also might not be a bad idea to mention SATA-Unsupported there as well:
* https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/troubleshooting.html#waiting-for-root-device-or-prohibited-sign-error

Please also verify you're fixed your markdown, we have the fix-in script as mentioned in the contributing page:
* https://dortania.github.io/OpenCore-Install-Guide/CONTRIBUTING.html

**Comment by gmachina** [comment]
hi its working now
andalso added in anther relavent place

**Comment by khronokernel** [comment]
> hi its working now

Your linting still fails:
* https://travis-ci.com/github/dortania/OpenCore-Install-Guide/jobs/363414445

**Comment by gmachina** [comment]
i dont know web i only know c c++ and java
can you tell what linting is exactly

**Comment by gmachina** [comment]
it was a mistake ill just create a new pull request

**Pull request closed without merging**